addons/sale/controllers/portal.py

```diff
@@ -161,6 +161,9 @@ def portal_order_page(self, order_id, report_type=None, access_token=None, messa
             body = _('Quotation viewed by customer %s') % order_sudo.partner_id.name
             _message_post_helper('sale.order', order_sudo.id, body, token=order_sudo.access_token, message_type='notification', subtype="mail.mt_note")
 
+        action_xmlid = 'sale.action_quotations_with_onboarding' \
```

Comment:
proper way to do this (because i don't like introspecting to see if the model has a field from another module):
1. define a method `get_portal_return_action` (or another meaningful name) on the `sale.order` model in `sale` that returns the default sale action
2. extend that method in `sale_renting` to return the correct method depending on the `is_rental_order` field
3. use this method in the controller of the sale's portal instead of the ugly introspection

deal?